README.md

```diff
@@ -1,83 +1,165 @@
+[![](https://api.codeclimate.com/v1/badges/a39e55b85bfcc31204b9/test_coverage)](https://codeclimate.com/github/enter-at/lambda-handlers/test_coverage)
+[![](https://requires.io/github/enter-at/lambda-handlers/requirements.svg?branch=master)](https://requires.io/github/enter-at/lambda-handlers/requirements/?branch=master)
+[![](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
 
-An opinionated Python package that facilitates specifying AWS Lambda handlers including input validation, error handling and response formatting.
+An opinionated Python package that facilitates specifying AWS Lambda handlers.
 
-## Dependencies
+It includes input validation, error handling and response formatting.
 
-This project uses [pipenv](https://pipenv.readthedocs.io) to manage its dependencies
-and Python environment. You can install it by:
+
+## Getting started
+
+To use `lambda-handlers` you must first install it:
 
 ```bash
-pip install --user pipenv
+pip install lambda-handlers
 ```
 
-We recommend using a Python virtual environment for each separate project you do.
-For that, we suggest using [pyenv](https://github.com/pyenv/pyenv-installer).
+If you are going to use validation, you should choose between
+[Marshmallow](https://pypi.org/project/marshmallow/) or
+[jsonschema](https://pypi.org/project/jsonschema/).
 
-## Installation
+To install with one of these:
 
-For production, after you clone this repository,
-you can install this project plus dependencies with:
+```bash
+pip install 'lambda-handlers[marshmallow]'
+```
+
+or
 
 ```bash
-cd <clone_dest>
-make install
+pip install 'lambda-handlers[jsonschema]'
 ```
 
-## Development
+### Quickstart
 
-For development you should also install the development dependencies,
-so run instead:
+By default the `http_handler` decorator makes sure of parsing the request body
+as JSON, and also formats the response as JSON with:
+   - an adequate statusCode,
+   - CORS headers, and
+   - the handler return value in the body.
 
-```bash
-cd <clone_dest>
-make install-dev
+```python
+from lambda_handler import http_handler
+
+@http_handler()
+def handler(event, context):
+    return event['body']
 ```
 
-This will install all dependencies and this project in development mode.
 
+### Examples
 
-### Testing
+Skipping the CORS headers default and configuring it.
 
-We use [tox](https://tox.readthedocs.io/en/latest/) to run the code checkers.
-Run it with:
+```python
+from lambda_handler import http_handler
+from lambda_handlers.response import cors
 
-```bash
-tox
+@http_handler(
+    cors=cors(origin='localhost', credentials=False),
+)
+def handler(event, context):
+    return event['body']
 ```
 
-You can also run the tasks separately with `make`:
 
-```bash
-make test
+Using jsonschema to validate a the input of a User model.
+
+```python
+from lambda_handler import validators, http_handler
```

Comment:
example would not work as `typing` import is missing

Reply:
you're right! done